docs/sources/articles/baseimages.md

```diff
@@ -41,15 +41,17 @@ GitHub Repo:
  - [Debian / Ubuntu](
    https://github.com/docker/docker/blob/master/contrib/mkimage-debootstrap.sh)
 
-## Creating a simple base image using `scratch`
+## Creating a simple base image using 'scratch'
```

Comment:
Thank you for the contribution.  I'd remove the markdown from the heading --- it looks odd.  Also, I think your structure can be tightened and emphasize the uniqueness by rearranging.

---> https://gist.github.com/moxiegirl/7a2b3ab626945b43ce1b

## Create a minimal image from scratch

Docker provides a special, official `scratch` image. Unlike other images, you cannot pull the `scratch` image and its name is reserved by the TagStore. The
`scratch` image was created from a empty `tar` file as follows:

    $ tar cv --files-from /dev/null | docker import - scratch

Use this image to create new minimal base images `FROM scratch`, for example:

    FROM scratch
    COPY true-asm /true
    CMD ["/true"]

The `Dockerfile` above is from an extremely minimal image - [tianon/true](
https://github.com/tianon/dockerfiles/tree/master/true).